frontend/src/scenes/data-warehouse/settings/source/Schemas.tsx

```diff
@@ -344,7 +352,7 @@ export const SchemaTable = ({ schemas, isLoading }: SchemaTableProps): JSX.Eleme
                                                                 }}
                                                                 status="danger"
                                                             >
-                                                                Resync
+                                                                Drop table and resync
```

Comment:
We're using both "drop" and "delete" - let's use one or the other for both options, they're doing the same thing (deleting their table from PostHog)